src/commands/list.ts

```diff
@@ -49,14 +49,23 @@ export const listCommand = {
           networkSettings,
           false,
         )
-        data[chalk.whiteBright.underline(dispute.id)] = disputeEntry
+        data[dispute.id] = disputeEntry
       }),
     )
     spinner.stop()
 
+    // sort disputes
+    const findDisputeById = (id: string) => disputes.find(d => d.id === id)
+    const orderedDisputeIds = Object.keys(data).sort((d1, d2) =>
```

Comment:
Something I was thinking is, we could make it so `populateEntry` fills an object but it doesn't necessarily have to be coupled to the display format, so that we could add the ID to the object and then have a function to format/display the disputes.

Reply:
We could refactor that in a different PR.